tutorials/20_REST_API.ipynb

```diff
@@ -0,0 +1,485 @@
+    "      embedding_dim: 768\n",
+    "  - name: Retriever \n",
+    "    type: EmbeddingRetriever\n",
+    "    params:\n",
+    "      document_store: DocumentStore\n",
+    "      embedding_model: sentence-transformers/multi-qa-mpnet-base-dot-v1\n",
+    "      model_format: sentence_transformers\n",
+    "      top_k: 5\n",
+    "  - name: FileTypeClassifier\n",
+    "    type: FileTypeClassifier\n",
+    "  - name: TextConverter \n",
+    "    type: TextConverter\n",
+    "  - name: PDFConverter\n",
+    "    type: PDFToTextConverter\n",
+    "  - name: Preprocessor\n",
+    "    type: PreProcessor\n",
+    "    params:\n",
+    "      split_by: word \n",
+    "      split_length: 768 \n",
+    "      split_overlap: 50 \n",
+    "      split_respect_sentence_boundary: True \n",
+    "\n",
+    "pipelines:\n",
+    "  - name: query\n",
+    "    nodes:\n",
+    "      - name: Retriever\n",
+    "        inputs: [Query]\n",
+    "  - name: indexing\n",
+    "    nodes:\n",
+    "      - name: FileTypeClassifier\n",
+    "        inputs: [File]\n",
+    "      - name: TextConverter\n",
+    "        inputs: [FileTypeClassifier.output_1]\n",
+    "      - name: PDFConverter\n",
+    "        inputs: [FileTypeClassifier.output_2]\n",
+    "      - name: Preprocessor\n",
+    "        inputs: [TextConverter, PDFConverter]\n",
+    "      - name: Retriever\n",
+    "        inputs: [Preprocessor]\n",
+    "      - name: DocumentStore\n",
+    "        inputs: [Retriever]\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "When you restart the Haystack API, REST API is going to load the new pipeline and the pipeline will be ready to use. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Voilà! Make a new query!\n",
+    "\n",
+    "This query should retrieve documents about _\"climate in Scandinavia\"_."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%bash\n",
+    "\n",
+    "curl -X 'POST' \\\n",
+    "  'http://localhost:8000/query' \\\n",
+    "  -H 'accept: application/json' \\\n",
+    "  -H 'Content-Type: application/json' \\\n",
+    "  -d '{\n",
+    "  \"query\": \"climate in Scandinavia\",\n",
+    "  \"params\": {}\n",
+    "}'"
+   ]
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3.9.6 64-bit",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.9.6"
+  },
+  "orig_nbformat": 4,
+  "vscode": {
+   "interpreter": {
+    "hash": "31f2aee4e71d21fbe5cf8b01ff0e069b9275f58929596ceb00d14d90e3e16cd6"
+   }
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 2
+}
```

Comment:
I'd add a paragraph summing up what they have learned/built/achieved in this tutorial. Maybe some links they may want to check, additional info, etc.